### PR TITLE
Import recent changes from mspnp/reference-architectures

### PR DESCRIPTION
**Pull request by hallihan**
Import recent changes from https://github.com/mspnp/reference-architectures/tree/master/data/enterprise_bi_sqldw

### CODE REVIEW

File: azure/templates/azure-resources-deploy.json [hunk 1]
```diff
@@ -103,16 +95,21 @@
...
+                        "tier": "DataWarehouse"
+                    },
+                    "kind": "v12.0,user,datawarehouse,gen2",
                     "properties": {
-                        "edition": "DataWarehouse",
-                        "requestedServiceObjectiveName": "[parameters('dwPerformanceTier')]",
-                        "collation": "[variables('collation')]"
+                        "collation": "[variables('collation')]",
+                        "maxSizeBytes": 263882790666240,
+                        "catalogcCllation": "[variables('collation')]"
```
Comment (ckittel) [inline azure/templates/azure-resources-deploy.json:112]: typo? `catalogcCllation`

**Review by ckittel — CHANGES_REQUESTED** [review]

File: azure/templates/azure-resources-deploy.json (hunk 1 above)
Comment (hallihan) [reply azure/templates/azure-resources-deploy.json:112]: I'm guessing so, @magrande this change was from https://github.com/mspnp/reference-architectures/commit/f4b902df333c7534a8faf31cff3122e0aedcde50 - feel free to weigh in.  I'll also look at the ARM docs to see if it's clear there.

Comment (hallihan) [reply azure/templates/azure-resources-deploy.json:112]: Looks like it's supposed to be "catalogCollation" <https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/servers/databases>, I'll update.

**Review by ckittel — APPROVED** [review]
🚀

**Pull request merged**